Recipes/InheritedTypeJsonConverter/InheritedTypeJsonConverter.cs

```diff
@@ -347,7 +414,11 @@ public override object ReadJson(JsonReader reader, Type objectType, object exist
         /// <inheritdoc />
         public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
         {
-            throw new NotImplementedException("not yet implemented");
+            string typeName = value.GetType().FullName + ", " + value.GetType().Assembly.GetName().Name;
```

Comment:
Is FullName not enough? Will the assembly name in here cause our types to break when a version is updated?